js/weekView.js

```diff
@@ -1,7 +1,204 @@
-import { isSameDate, isSameWeek, getWeekDates, selectedDate, getToday, HOUR_COUNT } from "../js/dateManipulation.js"
+import { isSameDate, isSameWeek, getWeekDates, getToday, HOUR_COUNT, LOCALE } from "../js/dateManipulation.js"
+import {selectedDate, storageState} from './state.js'
+import {findByHour} from './database.js'
+import { displayModal } from "./event.js"
+
+const EVENTBUBBLE_OFFSET = 25
+
+const sortByKey = (objectArray, key) => {
+    return objectArray.sort( (a,b) => {
```

Comment:
Return is optional. You can further shorten function by writting it like this:

const sortByKey = (........) => objectArray.sort........